drivers/regulator/regulator.c

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
This file should be in a separate PR

Reply:
It is. That's the commit from #114. It will only show up here until that PR is merged.